functions/Set-DbaMaxDop.ps1

```diff
@@ -97,19 +97,25 @@ function Set-DbaMaxDop {
         [switch]$AllDatabases,
         [switch]$EnableException
     )
+    begin {
+        if ($MaxDop -eq -1) {
+            $UseRecommended = $true
+        }
+    }
 
     process {
         if ((Test-Bound -Parameter Database) -and (Test-Bound -Parameter AllDatabases) -and (Test-Bound -Parameter ExcludeDatabase)) {
```

Comment:
@wsmelton - should this be changed to use ParameterName instead of Parameter? Also can I group this together like this?
```suggestion
        if ((Test-Bound -ParameterName Database, AllDatabases, ExcludeDatabase)) {
```

Reply:
If you would like, might as well make it consistent...